src/paperless/settings.py

```diff
@@ -880,14 +880,15 @@ def _parse_db_settings() -> dict:
 CELERY_BEAT_SCHEDULE_FILENAME = os.path.join(DATA_DIR, "celerybeat-schedule.db")
 
 # django setting.
+_REDIS_KEY_PREFIX = os.getenv("PAPERLESS_REDIS_PREFIX", "")
```

Comment:
This is used a few places above, I would move it to after https://github.com/paperless-ngx/paperless-ngx/blob/0d816af5d9eacf70de949f9f212ad68418df898c/src/paperless/settings.py#L432 and update other uses as needed